desc/protoparse/parser.go

```diff
@@ -840,7 +840,7 @@ func aggToString(agg []*aggregateEntryNode, buf *bytes.Buffer) {
 			aggToString(v.elements, buf)
 		} else {
 			buf.WriteString(": ")
-			elementToString(v, buf)
+			elementToString(a.val, buf)
```

Comment:
Pass `a.val.value()` and then you can omit the added case in `elementToString` below

Reply:
Done.